public/index.html

```diff
@@ -40,5 +40,6 @@ <h2 class="visually-hidden">Trip events</h2>
           </section>
       </div>
     </main>
+  <script src="/bundle.js"></script>
```

Comment:
```suggestion
  <script src="./bundle.js"></script>
```
Это чтобы в опубликованной версии путь правильно определялся.
И для стилей, где-то выше, сделай также, пожалуйста.